sequelizeDatabase/sequelFunctions.js

```diff
@@ -167,6 +167,37 @@ deleteCompany = (CompanyID) => {
         })
     })
 }
+
+retrieveCurrentEmployeesOfCompany = (CompanyID) => {
+    return new Promise((resolve,reject)=>{
+        getAllIndividuals().then((people)=>{
+            Promise.all(people.map((person)=>{
```

Comment:
Takes an array of promises, when all promises in the array finish, it returns a promise of the compiled results to the .then() statement